assets/helpers/page/page.js

```diff
@@ -23,7 +24,7 @@ import type { Action } from './pageActions';
 export type CommonState = {
   campaign: ?Campaign,
   referrerAcquisitionData: ReferrerAcquisitionData,
-
+  otherQueryParams: [string, string][],
```

Comment:
What would this type be? An array of tuples?

Reply:
If it is an array I would use the Array notation to make it more readable: https://flow.org/en/docs/types/arrays/